### DISCUSSION TITLE
Issue with Vite in build mode

### DISCUSSION THREAD

**Discussion by dblythy** [quasarframework/quasar#15009] [CLI - PWA mode]
Hi,

I am building a Quasar app and I am migrating to Vite. All works well, except once I serve the generated pages as a SPA, I get `TypeError: (0 , _decode2.default) is not a function` when my button action fires. It seems that the module `decode2` doesn't load correctly when the button is clicked.

Decode is a `export` from a `node_module` - Parse JS SDK

Seems to have no issues in dev mode - only occurs once I generate the build files and serve them.

Does anyone have any idea how to fix this?

Thanks!

**Comment by dblythy** [comment]
Does anyone have any pointers for how to resolve this 😊

**Comment by suleimanas** [comment]
with Parse I think you need to use the minified import 
`import Parse from 'parse/dist/parse.min.js';`

**Reply by dblythy** [reply]
Yeah, that works too, but the types drop off. I'm trying to create a more solid solution. It seems that Parse JS SDK compiles with babel `export default` as: `{ default: ..., __esModule: true }` which seems to be problematic with Rollup/Vite in build mode. I'm wondering how to fix it on the upstream. Perhaps the question is better posted there.
